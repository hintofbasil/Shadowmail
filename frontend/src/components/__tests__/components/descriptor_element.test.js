import React from 'react';
import renderer from 'react-test-renderer';

import DescriptorELement from '../../descriptor_element';

describe('render', () => {
  it('renders successfully', () => {
    const tree = renderer
      .create(
        <DescriptorELement
          title="title"
          symbol="symbol"
          description={<span>description</span>}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
