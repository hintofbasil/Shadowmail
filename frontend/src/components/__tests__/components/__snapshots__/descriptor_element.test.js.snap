// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders successfully 1`] = `
<div
  className="four columns descriptor descriptor-three"
>
  <div
    className="descriptor-title"
  >
    title
  </div>
  <div>
    <i
      className="material-icons descriptor-icon"
    >
      symbol
    </i>
  </div>
  <div
    className="descriptor-text"
  >
    <span>
      description
    </span>
  </div>
</div>
`;
